external/pkg/docker-exec-ws/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package dockerExecWS provides a golang client implementation for the websocket protocol implemented in
+// the npm package "docker-exec-websocket-server". This is a websocket protocol for exposing "docker exec" to
+// a remote process.
+package dockerExecWS
```

Comment:
how about `package dockerexecws`, oh, that's not much better... Can we come up with a better package name or should we just give up?

See: https://blog.golang.org/package-names

Reply:
I'm mostly thinking we want to avoid problems with metalinter, I filed #171 to update the config and run more linters, but that shouldn't block this work :)